src/models/PolicyList.ts

```diff
@@ -63,13 +63,27 @@ declare interface PolicyList {
 /**
  * The PolicyList caches all of the rules that are active in a policy room so Mjolnir can refer to when applying bans etc.
  * This cannot be used to update events in the modeled room, it is a readonly model of the policy room.
+ *
+ * You can inform the PolicyList about new events in a room with the `updateForEvent`, this will eventually
```

Comment:
there's a lot of overlapping terminology here; does this mean "you can inform a PolicyList object about new events in its respective policy room [...]"?

Reply:
i've tried to make things a bit clearer and explicit